### PR TITLE
Vive HMD fixes to prevent user from becoming embedded in the floor.

### PR DESCRIPTION
**Pull request by hyperlogic**
There are two instances were vive HMD users we're likely to become embedded in the floor or teleported off into space.

* When the HMD was removed the user goes into away mode.  Coming back from away mode could cause avatar to be teleported downward by a meter or so.
* Starting interface without Steam VR running in the background, could cause the avatar to be teleported far from the original spawn location.

### CODE REVIEW

**Comment by hyperlogic** [comment]
test this please

**Comment by hyperlogic** [comment]
Test plan:
   * Start interface without SteamVR running in the background, you should not end up below the floor or far from your original spawn location.
   * Take off your headset then put it back on.  You should see the "paused" mode overlay.  
   * Press the trackpad on the hand controller to exit paused mode.  This should NOT cause your body to move downward and potentially be embedded in the floor.

**Comment by hyperlogic** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 90beb30e8a0d0bcb1953c8d88d5b4a5cf998713b available at: https://gist.github.com/7a7528f05424c8ce890b81c86549c444<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8282-90beb30.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8282-90beb30.dmg

File: plugins/openvr/src/OpenVrDisplayPlugin.cpp [hunk 1]
```diff
@@ -139,6 +152,12 @@ void OpenVrDisplayPlugin::resetSensors() {
     });
 }
 
+static bool isBadPose(vr::HmdMatrix34_t* mat) {
+    if (mat->m[1][3] < -0.2f) {
```
Comment (ZappoMan) [inline plugins/openvr/src/OpenVrDisplayPlugin.cpp]: this could use a comment I think...

**Comment by SkimiMission** [comment]
- [SUCCESS] Start interface without SteamVR running in the background, you should not end up below the floor or far from your original spawn location.
- [SUCCESS] Take off your headset then put it back on. You should see the "paused" mode overlay.
- [SUCCESS] Press the trackpad on the hand controller to exit paused mode. This should NOT cause your body to move downward and potentially be embedded in the floor.

Very smooth, nice work.

**Pull request merged**